test/integration/conftest.py

```diff
@@ -0,0 +1,44 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
There's a LICENSE in the top level, so we can probably remove these.

Reply:
All of them?

Reply:
It's what I've seen in a number of repos. It's not doing any harm, though, so it's up to you. :smile: